force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -322,17 +421,21 @@ public with sharing class FlowControlLoopsCollections {
      */
     public static Integer getSalary(String employeeName) {
         // Initialize a Map
-
+        Map<String,Integer> employeeSalary = new Map<String,Integer>();
         // Add key-value pairs to the Map
         // add John Doe => 50000
+        employeeSalary.put('John Doe', 50000);
         // add Jane Smith => 60000
+        employeeSalary.put('Jane Smith', 60000);
         // add Sam Brown => 55000
+        employeeSalary.put('Sam Brown', 55000);
         // add Alice Johnson => 65000
+        employeeSalary.put('Alice Johnson', 65000);
```

Comment:
Try using a static variable for employeeSalaries for questions 19 and 20 to reduce duplicates since the values are the same